### PR TITLE
Some API cleanup

### PR DESCRIPTION
**Pull request by mrossinek**
This performs most of the aspects of #35.

Looking at each commit individually might make the review process a bit easier in this case.

### CODE REVIEW

File: povm_toolbox/library/metadata/rpm_metadata.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ class RPMMetadata(POVMMetadata):
     pvm_keys: np.ndarray
     """The keys which associate a specific result sample with the corresponding RPM parameters.
 
-    Shape of `pvm_keys` is assumed to be ``(*pv.shape, num_batches, n_qubit)``,
+    Shape of `pvm_keys` is assumed to be ``(*pv.shape, num_batches, num_qubit)``,
```
Comment (timmintam) [inline povm_toolbox/library/metadata/rpm_metadata.py:29]: Small detail but here shouldn't it be `num_qubits` with an "s" ?

**Review by timmintam — APPROVED** [review]
LGTM 👍 only one tiny detail

File: povm_toolbox/library/metadata/rpm_metadata.py (hunk 1 above)
Comment (mrossinek) [reply povm_toolbox/library/metadata/rpm_metadata.py:29]: Nice job spotting this! :+1:

**Comment by mrossinek** [comment]
Okay, I accidentally included 3387f9115bdbf6739a7c4f240a6c73826e3381c0 in here. I guess its fine though.
This was my start at a follow-up docs clean-up PR...

**Review by timmintam — APPROVED** [review]

**Pull request merged**